cli/npm/installer/common/mod.rs

```diff
@@ -27,48 +29,56 @@ pub trait NpmPackageFsInstaller: std::fmt::Debug + Send + Sync {
   ) -> Result<(), JsErrorBox>;
 }
 
-pub trait NpmPackageExtraInfoProvider: std::fmt::Debug + Send + Sync {
```

Comment:
It seemed we didn't really need this trait, so I removed it. We can add it back later if we need it.